labelbox/schema/dataset.py

```diff
@@ -182,10 +184,32 @@ def data_row_for_external_id(self, external_id):
         where = DataRow.external_id == external_id
 
         data_rows = self.data_rows(where=where)
-        # Get at most two data_rows.
-        data_rows = [row for row, _ in zip(data_rows, range(2))]
+        # Get at most `limit` data_rows.
+        data_rows = [row for row, _ in zip(data_rows, range(limit))]
```

Comment:
What about 

`list(islice(data_rows, 0, limit))`

https://docs.python.org/3/library/itertools.html#itertools.islice

According to itertools that seems to be most idiomatic

Reply:
changed